src/client.rs

```diff
@@ -147,13 +147,15 @@ impl FtxClient {
             req
         };
 
+        println!("{:?}", req);
```

Comment:
If you want to leave this in code, best to use the log::debug! macro

Reply:
Left it in there, changed to logging.